src/lib/components/TokenSelect/index.tsx

```diff
@@ -81,7 +86,7 @@ export default function TokenSelect({ value, collapsed, disabled, onSelect }: To
       <TokenButton value={value} collapsed={collapsed} disabled={disabled} onClick={() => setOpen(true)} />
```

Comment:
same as above - or `selectedCurrency` or something

Reply:
see above - `value` is intentional